### DISCUSSION TITLE
An unknown action created be me

### DISCUSSION THREAD

**Discussion by panla** [fastapi/fastapi#6246] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
no
```

### Description

now , i see an unknown action in this project (fastapi) created by me
![深度截图_选择区域_20220702091313](https://user-images.githubusercontent.com/41326348/176981372-20fff0e4-95ec-4688-b84c-245a20e78144.png)

### Operating System

Linux

### Operating System Details

deepin

### FastAPI Version

3.9.12

### Python Version

0.78.0

### Additional Context

no

**Comment by JarroVGIT** [comment, marked as answer]
This project uses a custom `Issue Manager`, which is an automated job that runs after every issue that is opened. You can see here the source code of that manager: https://github.com/tiangolo/issue-manager. Basically, it closes any issue without a response for 10 days by adding a delayed response.

The action that is failing, is due to the rate limiter of Github. However, there is a scheduled job that counters this; the delayed response on your issue was created in this run: https://github.com/tiangolo/fastapi/actions/runs/2599758008
